module-metadata.json

```diff
@@ -1,12 +1,142 @@
 {
   "path": ".",
-  "variables": {},
-  "outputs": {},
+  "variables": {
+    "plan": {
+      "name": "plan",
+      "type": "string",
+      "description": "Plan for the CIS instance. Standard-next or trial.",
```

Comment:
Sync the changes in `variables.tf` in the module-metadata.file

Reply:
done